pandas/core/strings/accessor.py

```diff
@@ -2409,6 +2419,17 @@ def get_dummies(self, sep: str = "|"):
         ----------
         sep : str, default "|"
             String to split on.
+        prefix : str, list of str, or dict of str, default None
```

Comment:
Can't users just prefix the columns by calling `.rename` after the call to get_dummies?